pkg/media/pcmlocaltrack.go

```diff
@@ -211,6 +209,26 @@ func (t *PCMLocalTrack) processSamples() {
 	t.opusWriter.Close()
 }
 
+func (t *PCMLocalTrack) setMuted(muted bool) error {
+	if t.closed.Load() {
+		return errors.New("track is closed")
+	}
+
+	// We will continue to write silence on mute to not mess
+	// up the RTP timestamps. An edge case here is that the mute
+	// value will be checked on every tick (10ms), so any new data on unmute
+	// might be delayed by at most 10ms, which is not much and keeps RTP/RTCP
+	// in sync. The packets will be dropped by the SFU anyway.
```

Comment:
I am guessing this comment is outdated. The mute check in processSamples() has been removed. You can just mention that on mute, the pending samples are dropped.

Reply:
Yeah, I removed the mute check because your code would already send silence there, no need to check if track is muted or not. We need to continue to send silence to not mess up the RTP timestamps. So the comment is not outdated. 

So the behavior is: pending samples are dropped and the track writes silence to keep RTP timestamps in check. But these silence packets will be dropped by the SFU anyway.

Reply:
The comment says
```
    // An edge case here is that the mute
    // value will be checked on every tick (10ms), so any new data on unmute
    // might be delayed by at most 10ms, which is not much and keeps RTP/RTCP
    // in sync. The packets will be dropped by the SFU anyway.
```

I read that as `processSamples` checks for mute explicitly which it is not doing. And can probably replace 10ms with the constant name (if somebody changes that constant to 20ms, this comment will be outdated). And maybe drop SFU reference, that is behaviour of some remote app which may change at any time. I think the main point here is that this keep sending packets even when muted to maintain RTP timestamps and possibly leave it at that.

Reply:
Understood, thanks!